udata/templates/organization/search-result.html

```diff
@@ -8,8 +8,8 @@
                 width="70" height="70">
         </div>
         {{ badge_if_certified(organization) }}
-        <div class="result-body ellipsis-dot">
-            <h4 class="result-title">{{ organization.name }}</h4>
+        <div class="result-body ellipsis">
```

Comment:
Why not `ellipsis` on `h4` below and `clamp-2` on the following `div` following the same pattern as others?